---
title: "ChromoTrace: Computational reconstruction of 3D chromosome configurations for super-resolution microscopy."
collection: publications
permalink: /publication/2018-03-03-paper-title-number-3
excerpt: 'This paper is about reconstructing the 3D structure from super resolution microscopy images.'
date: 2018-03-03
venue: 'PLoS Computational Biology'
paperurl: 'https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006002'
---

The 3D structure of DNA in the nucleus is known to be important for many aspects of DNA function, such as how gene expression is regulated. However, current techniques to localise or determine 3D DNA structure are often indirect. The advent of super-resolution microscopy, at a resolution of 20 nm or better can directly visualize fluorescent probes bound to specific DNA in the nucleus. However it is not trivial to associate how many specific stretches of DNA lie relative to each other, making reliable and precise 3D mapping of large stretches of the genome difficult. Here, we propose a method that leverages the fact that we know the sequence of the genome and the resolution of the super-resolution microscope. Our method, ChromoTrace, uses a computer science data structure, suffix trees, that allow one to simultaneous search the entire genome for specific sub-sequences. To show that our method works, we build a simulation scheme for simulating DNA as ensembles of polymer chains in a nucleus and explore the sensitivity of our method to different types of error. ChromoTrace can robustly and accurately reconstruct 3D paths in our simulations.

